IdentityCore/src/cache/metadata/MSIDMetadataCache.m

```diff
@@ -148,7 +153,7 @@ - (MSIDAccountMetadataCacheItem *)accountMetadataCacheItemWithKey:(MSIDCacheKey
     }
     
     dispatch_barrier_async(_synchronizationQueue, ^{
-        _memoryCache[key] = item;
+        self.memoryCache[key] = item;
```

Comment:
there's now inconsistency between here and line 132. Do you know why it is different?

Reply:
I just have forgotten to apply self. prefix, and XCODE wasn't throwing the warning.

Reply:
Do you know why?

Reply:
As discussed during the scrum meeting, it appears that dispatch_barrier_sync doesn't enforce the requirement of appending self. or self->_ prefix within a block statement when you enable *CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF*